spec/obfuscator_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+require 'kantox-obfuscator'
+
+RSpec.describe Kantox::Obfuscator do
+  it 'obfuscates strings by default' do
+    expect(subject.call('1234567890')).to eq('12xxxx7890')
+  end
+
+  it 'obfuscates numbers by default' do
+    expect(subject.call(1_234_567_890)).to eq('12xxxx7890')
+    expect(subject.call(1_234_567_890.12)).to eq('12xxxxxxx0.12')
```

Comment:
Please do test what is less obvious, not what is as simple that literally might be tested inside your brain VM.
```ruby
expect(subject.call(1)).to WHAT?
expect(subject.call("1")).to WHAT?
expect(subject.call(nil)).to WHAT?
expect(subject.call({})).to WHAT?

```